src/components/RegionItem/RegionItem.style.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { COLOR_MAP } from 'common/colors';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+
+export const CircleIcon = styled(FiberManualRecordIcon)<{ iconColor: string }>`
+  color: ${({ iconColor }) => iconColor};
+  font-size: 1rem;
+`;
```

Comment:
Consider renaming the property `iconColor` to be `$iconColor` so it's not passed down to the wrapped `FiberManualRecordIcon`, this will avoid this warning in the console

![image](https://user-images.githubusercontent.com/114084/105289040-e1ee0480-5b6c-11eb-94b6-93e1d8f727b7.png)